src/util/getUrl.ts

```diff
@@ -3,7 +3,7 @@ import envVars from "../config/env";
 export const getGitHubConfigurationUrl = (
 	jiraHost
 ): string =>
-	`https://${jiraHost}/plugins/servlet/ac/com.github.integration.${envVars.INSTANCE_NAME}/github-post-install-page`;
+	`${jiraHost}/plugins/servlet/ac/com.github.integration.${envVars.INSTANCE_NAME}/github-post-install-page`;
```

Comment:
Also we should test it. Another thing which concerned me is missing colon after http in the redirect URL:
`https://https//kabakumov.atlassian.net/plugins/servlet/ac/com.github.integration.production/github-post-install-page`

Reply:
the missing colon is normal as the browser will try to 'parse' the URL and you can't have a colon in the domain, so the browser removes it.

Reply:
Also, even if I fix the URL manually, I'm redirected to the list of installed orgs, which the current org wasn't installed yet... Shouldn't it rather redirect to `https://github.stg.atlassian.com/github/configuration` ?

Reply:
@bgvozdev I think you might be confusing the flow, but I can see why because the function name isn't well described.  I'll create another PR to fix the name